.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+<!-- 
+Please make sure you have read the contribution guidelines https://github.com/grafana/k6/blob/master/CONTRIBUTING.md as well as the
+the code of conduct https://github.com/grafana/k6/blob/master/CODE_OF_CONDUCT.md before opening a PR.
+-->
+
+## What?
+
+<!-- A description of the changes this PR brings to the documentation. -->
+
+## Checklist
+
+Please fill in this template:
+- [ ] I have used a meaningful title for the PR.
+- [ ] I have described the changes in made in the "What?" section above.
+- [ ] I have performed a self-review of my changes.
+- [ ] I have run the `make docs` command locally and verified that the changes look good.
+
+Select one of these and delete the others:
+
+If updating the documentation of the most recent release of k6: 
+- [ ] I have made my changes in the `docs/sources/v{most_recent_release}` folder of the documentation.
+- [ ] I have reflected my changes in the `docs/sources/next` folder of the documentation.
+- [ ] I have reflected my changes in the folders of the previous k6 versions of the documentation.
+
+If updating the documentation of the next release of k6:
+- [ ] I have performed a self-review of my changes.
+- [ ] I have made my changes in the `docs/sources/next` folder of the documentation.
+- [ ] I have reflected my changes in the `docs/sources/v{most_recent_release}` folder of the documentation.
```

Comment:
```suggestion
```

I think for updates to the next version of k6, people just need to update the `docs/sources/next`.